R/tokens.R

```diff
@@ -178,16 +178,67 @@ tokens.character <- function(x, ...) {
 #' @rdname tokens
 #' @export
 #' @noRd
-tokens.corpus <- function(x, ..., include_docvars = TRUE) {
+tokens.corpus <- function(x, ..., hash = TRUE, include_docvars = TRUE) {
```

Comment:
Interesting - I thought we would get errors in the CHECK if not all the signatures are the same.